components/partials/markets/market-row.vue

```diff
@@ -182,6 +213,28 @@ export default Vue.extend({
       return volumeInUsd.toFormat(2)
     },
 
+    abbreviatedVolumeInUsdToFormat(): string {
```

Comment:
Extract this to a utility function, it can be useful for another use case in the future.